test/unit/uk/gov/hmrc/tariffclassificationfrontend/controllers/RulingControllerSpec.scala

```diff
@@ -45,10 +45,11 @@ class RulingControllerSpec extends WordSpec with Matchers with GuiceOneAppPerSui
   private val casesService = mock[CasesService]
   private val fileService = mock[FileStoreService]
   private val mapper = mock[DecisionFormMapper]
+  private val decisionForm = new DecisionForm(new CommodityCodeConstraints(new CommodityCodeService))
```

Comment:
Would it not be more normal to mock the CommodityCodeService?

Reply:
Done